src/System.Data.SqlClient/tests/ManualTests/SQL/LocalDBTest/LocalDBTest.cs

```diff
@@ -62,8 +62,7 @@ public static void LocalDBNotSupportedOnUapTest()
             builder.IntegratedSecurity = true;
             builder.ConnectTimeout = 2;
 
-            string errorMsg = SystemDataResourceManager.Instance.LocalDBNotSupported;
```

Comment:
Are we consistently removing such verification? Is this overload of the assert helper still needed?